launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.html

```diff
@@ -2,17 +2,17 @@
     <h2>Enter the Name of Your Podcast</h2>
     <mat-form-field appearance="outline">
         <mat-label>Podcast Title</mat-label>
-        <input matInput placeholder="Ex. This American Life">
+        <input matInput name="title" placeholder="Ex. This American Life">
     </mat-form-field>
     <br />
 
     <h2>Provide Your mp3 Link</h2>
     <mat-form-field appearance="outline">
         <mat-label>mp3 Link</mat-label>
-        <input matInput placeholder="Paste the mp3 link here">
+        <input matInput name="mp3Link" placeholder="Paste the mp3 link here">
     </mat-form-field>
```

Comment:
Just for my knowledge, is it always an mp3? 

Super nit: If not we can name the input something more generic like sourceFileUrl or something like that.

Reply:
yes, this input is always going to be a link to an mp3 file.